### ISSUE TITLE
[Documentation] Update readme to advise users trying howdy on sddm+kde

### ISSUE THREAD

**Issue by lcuis** [Boltgolt/howdy#713]
Thanks for howdy, it is certainly great for those who can use it! I was able to use it on terminal sessions and it worked fantastically!

Like some others, I have sddm+kde and have been running into issues with howdy.

My case was a bit specific in the sense that I couldn't log into kde anymore after installing howdy.
I was able to type a password in the login screen but there was no way to validate it. The return keys had no effect.
Plus, howdy was activating the camera and recognizing me but not opening a kde session.
Selecting another environment than kde didn't change anything.
I found nothing particularly exciting in the logs.

The only way I found to recover from this situation was to use the key combination "ctrl+alt+f2" and log into a terminal session. There, again, howdy recognized me.
Uninstalling howdy and rebooting made it possible to recover kde in a classical way.

As **I understand the issue is not with howdy** but with sddm+kde, maybe a way to manage expectations and provide users with sufficient information on how to solve a likely issue would be to update the howdy readme file with the following for example?

### sddm+kde users

Dear sddm+kde users, it is possible that you encounter difficulties logging in once howdy is installed. It is likely due to issues with your environment. If you are stuck on the sddm login invite without the possibility to log in, please follow the following steps:
1. Press the keys combination "ctrl+alt+f2". Where f2 can be replaced by any other f key where a terminal session is available.
2. Log in with a user who has the possibility to uninstall howdy or a su/sudo access.
3. Uninstall howdy.
4. Reboot.

Hopefully, all your problems are gone.
If howdy is more important to you than your choice of window manager, you can also try another option than sddm+kde for better results.

**Comment by nalin29** [comment]
I see the same behavior with kscreenlocker. The other stuff, SDDM/POLKIT are fine. it seems that this happens repeatedly when the compute auto locks. In fact howdy does seem to be working? Since in many cases after a long delay it will attempt to scan face after several minutes of blocking.

**Comment by lcuis** [comment]
Hi @nalin29 ,

Thanks for your comment. I see your issue #716 related to this problem.

I indeed have this package installed: `libkscreenlocker5`

Would you suggest for me to update the title of my issue and the content of the suggested advice to replace "sddm+kde" by "kscreenlocker"?
It looks like this package is a dependency for kde to run.

**Comment by lcuis** [comment]
I have noticed with issue #679 that my problem is likely to be solved once my KDE is updated to version 5.25.2 . It is currently version 5.24.6 . Looking forward to trying howdy again once my kde is updated!

**Comment by lcuis** [comment]
Using [this link](https://www.omgubuntu.co.uk/2022/08/kubuntu-22-04-users-can-now-upgrade-to-kde-plasma-5-25), I was able to update my kde version to 5.25.5 . Though, I still got the exact same symptoms using howdy. So, this solution doesn't apply to my situation sadly and my recommendation to update the readme with the advice for sddm+kde users seems to still be valid.

**Comment by nalin29** [comment]
Yeah, its unfortunate I have yet to find a solution. The dmesg does not help. Its random which makes it harder to test and debug I have just turned it off for kscreenlock which is unfortunate since it works so well.

**Comment by boltgolt** [comment]
I think adding that to the readme would be good for any users really, i've had my fair share of lockouts during development too. Maybe adding a failsafe to Howdy would be nice, like holding escape will skip Howdy in the PAM order

**Comment by lcuis** [comment]
Hi @boltgolt ,

Yes, the esc holding solution would be wonderful!

In that case, the readme advice could be something like:

## If you cannot login

It can happen that you get stuck at the login invite. In that case, holding the escape key for 5 seconds will skip Howdy in the PAM order and let you use an old style login solution.

**Comment by nunosasilva** [comment]
Hello! :)
I know this issue is a few months old, anyway i had similar login problems with kde neon and stumbled on this.
I think that the culprit are the wallets, that don't like not receiving a password. What works for me is commenting out the lines:

-auth   optional        pam_gnome_keyring.so
-auth   optional        pam_kwallet5.so

...in /etc/pam.d/sddm

Hope it helps :)

**Comment by lcuis** [comment]
Hi @nunosasilva ,

Thanks for this solution.
It worked for me as I was able to log into KDE with howdy after editing the sddm file as you mentioned.
However, once logged in, I have to enter twice the password for the keyring/wallet.
Maybe I misconfigured something else.
